metricbeat/module/kubernetes/_meta/config.yml

```diff
@@ -6,3 +6,8 @@
     - system
     - volume
   hosts: ["localhost:10255"]
+  #ssl.certificate_authorities: ["/etc/pki/root/ca.pem"]
+  #ssl.certificate: "/etc/pki/client/cert.pem"
+  #ssl.key: "/etc/pki/client/cert.key"
```

Comment:
ssl options were not introduced on #7134, but I guess that it is something related.